torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -981,13 +990,16 @@ def elastic(
 elastic_transform = elastic
 
 
-def _center_crop_parse_output_size(output_size: List[int]) -> List[int]:
+def _center_crop_parse_output_size(output_size: Union[int, List[int]]) -> List[int]:
     if isinstance(output_size, numbers.Number):
         return [int(output_size), int(output_size)]
-    elif isinstance(output_size, (tuple, list)) and len(output_size) == 1:
-        return [output_size[0], output_size[0]]
+    elif isinstance(output_size, (tuple, list)):
+        if len(output_size) == 1:
+            return [output_size[0], output_size[0]]
+        else:
+            return list(output_size)
     else:
-        return list(output_size)
+        raise ValueError("Invalid output_size")  # Needed for jit
```

Comment:
Normally, we should never hit this exception. After just changing the output type n the signature, neither mypy nor JIT can understand that the previous `else` statement activates only when in list. This is because it doesn't recognise that the first `if` statement activates for integers. So we have to refactor it and explicitly assert that we are dealing with lists.

Reply:
This is no longer strictly necessary. Perhaps it looks cleaner? Let me know if I should revert

Reply:
I would revert since it is dead code and can never be reached.